components/extractors/storage/aws-s3/index.md

```diff
@@ -65,8 +65,8 @@ Select `Role` as the **Login Type**. Create a role in your AWS account using the
 
  - Go to the [IAM Console](https://console.aws.amazon.com/iam/home?#/roles) and click **Create role**. Then click **Another AWS account**.
  - For **Account ID**
-   - use `206948715642` for stack `connection.europe-west3.gcp.keboola.com` 
-   - use `147946154733` for all other stacks
+   - use `147946154733` for stacks `connection.keboola.com`, `connection.eu-central-1.keboola.com`, `connection.north-europe.azure.keboola.com`.
+   - use `206948715642` for all other stacks.
```

Comment:
Přijde mi lepšít to prohodit. Tzn. stacky které používají legacy account vyjmenovat a pro všechny nové už bude správně v novém accountu.